pbr/config/scene_config.py

```diff
@@ -22,11 +22,11 @@
 
 resources = {
     "robot": {
-        "mesh_path": path.abspath(path.join(res_path, "robot", "NUgus.fbx")),
+        "mesh_path": path.abspath(path.join(res_path, "robot", "NUgus_esh.fbx")),
         "texture_path": path.abspath(path.join(res_path, "robot", "textures")),
-        "kinematics_path": path.abspath(path.join(res_path, "robot", "NUgus.json")),
+        "kinematics_path": path.abspath(path.join(res_path, "robot", "NUgus_esh.json")),
         "kinematics_variance": 0.5,  ## Determines how much variance the random poses will have
-        "mask": {"index": 3, "colour": (0, 0, 1, 1)},
+        "mask": {"index": 6, "colour": (0, 0, 1, 1)},
```

Comment:
What's up with this change? I've tested the branch on two different computers and this change seems to causing problems on one of them.

Reply:
Can you elaborate on what the problems are?

Reply:
@ysims can you pls confirm if you're experiencing the same error as Joe? Thanks in advance

Reply:
ooh is it the colour of the robots in the mask? I cant even remember why i made this change. I'll check the outputs in a moment

Reply:
Idx reverted to 3. Thanks for the pickup!